weblate/static/loader-bootstrap.js

```diff
@@ -850,71 +435,6 @@ $(function () {
         }
     }
 
-    /* Translation editor */
-    Mousetrap.bindGlobal(
-        ['alt+enter', 'mod+enter'],
-        submitForm
-    );
```

Comment:
This is used to submit form anywhere, it's not specific to the editor.

Reply:
Do you have any pointers where? I understand this is needed in the editor because it's a textarea, but any other regular forms can/should be natively submitted by pressing <kbd>Enter</kbd> (the `submit` event can be captured to attach XHR behavior).

Reply:
For example, settings or addon configuration forms have textareas and it's useful there as well to submit with Ctrl+Enter.